contrib/agent/build.gradle

```diff
@@ -30,6 +30,9 @@ def agentBootstrapClasses = agentBootstrapPackageDir + '**'
 def agentRepackaged = "${agentPackage}.deps"
 
 dependencies {
+  compileOnly libraries.grpc_context
+  testCompile libraries.grpc_context
+  compile group: 'com.google.code.findbugs', name: 'annotations', version: '3.0.1u2'
```

Comment:
I'll make a separate PR for the findbugs version. The root buildscript uses two artifacts, the jsr305 one (3.0.0) and another one for the tool (3.0.1). I guess there's no reason for using a different version.

The jar referenced here is yet another one, for the SuppressFBWarnings annotation, which then should also use the same version.